docs/Examples/Basic.md

```diff
@@ -49,11 +49,12 @@ extension MyService: TargetType {
             return ["first_name": firstName, "last_name": lastName]
         }
     }
-
     var parameterEncoding: ParameterEncoding {
         switch self {
-        case .zen, .showUser, .showAccounts, .updateUser:
+        case .zen, .showUser, .showAccounts:
             return URLEncoding.default // Send parameters in URL
+        case .updateUser:
+            return URLEncoding.queryString // Send parameters in URL
```

Comment:
Let's specify in comment that it would **always** send parameters in URL here, and above will depend on a http method (`GET`, `HEAD` and `DELETE` being in URL, and in body otherwise)